src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no, initial-scale=1.0,
+          maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
Need to fix everywhere
<img width="1021" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/d5ea8966-e93c-4408-9056-02068d632088">